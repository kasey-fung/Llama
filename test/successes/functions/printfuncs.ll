void main():
    int i
    float f
    string s
    char c
    bool b

    i = 20
    f = 30.0
    s = "Hello World!"
    c = 'c'
    b = true

    /* print(int)  */
    print(i)
    print(1)

    /* print float  */
    print(f)
    print(1.25)

    /* print string */
    print(s)
    print("Hello world")

    /* print char */
    print('a')
    print(c)

    /* print bools */
    print(b)
    print(true)
