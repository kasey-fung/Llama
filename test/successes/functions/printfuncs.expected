20
1
30
1.25
"Hello World!"
"Hello world"
a
c
1
true